pkg/container/container.go

```diff
@@ -62,6 +71,21 @@ type Container interface {
 	// SelinuxLabel returns the container's SelinuxLabel
 	// it takes the sandbox's label, which it falls back upon
 	SelinuxLabel(string) ([]string, error)
+
+	// spec functions
+
+	// returns the spec
+	Spec() *generate.Generator
+
+	// SpecAddMount adds a mount to the container's spec
+	// it takes the destination, the mount type, the source type, the mount to remove, and the options
+	SpecAddMount(string, string, string, string, []string)
```

Comment:
1. Having the "mount to remove" argument seems questionable, especially since we already have a method that does exactly that.

2. I'd suggest to use variable names here, as the form of self-documentation.

IOW
```go
SpecAddMount(destination, mountType, sourceType string, options []string)
```

It reads way better than "string, string, string, string" :) and you can now omit the parameters description from the docstring.

Reply:
fixed !